externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/result/IncrementalResultFetchIterator.scala

```diff
@@ -125,7 +125,16 @@ class IncrementalResultFetchIterator(
         Thread.sleep(FETCH_INTERVAL_MS)
       }
     })
-    Await.result(future, resultFetchTimeout)
+    try {
+      Await.result(future, resultFetchTimeout)
+    } catch {
+      case e: TimeoutException =>
```

Comment:
We need a `case _ ` to cover other exceptions. Simply rethrow them would be good.